From 63cf1bb5da586dc1e66b5541ee49d84ef71c015c Mon Sep 17 00:00:00 2001
From: Jonathan Waldrep <waldrep@vt.edu>
Date: Fri, 22 Feb 2019 13:46:50 -0500
Subject: [PATCH 2/8] use anonymous pro

---
 config.def.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/config.def.h b/config.def.h
index 0e01717..6781b82 100644
--- a/config.def.h
+++ b/config.def.h
@@ -5,7 +5,7 @@
  *
  * font: see http://freedesktop.org/software/fontconfig/fontconfig-user.html
  */
-static char *font = "Liberation Mono:pixelsize=12:antialias=true:autohint=true";
+static char *font = "Anonymous Pro:pixelsize=18:antialias=true:autohint=true";
 static int borderpx = 2;
 
 /*
-- 
2.20.1

